code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -180,3 +189,235 @@
+	flags_inv = NONE
+	// slightly worse than normal cult robes
+	armor = list(MELEE = 30, BULLET = 30, LASER = 30,ENERGY = 30, BOMB = 15, BIO = 0, RAD = 0, FIRE = 0, ACID = 0)
+	pocket_storage_component_path = /datum/component/storage/concrete/pockets/void_cloak
+	///reference to the componenet so we dont have to call get_component every time we toggle hood
+	var/datum/component/storage/concrete/pockets/void_cloak/cloak
+	alternative_mode = TRUE
+
+/obj/item/clothing/suit/hooded/cultrobes/void/ToggleHood()
+	if(!iscarbon(loc))
+		return
+	var/mob/living/carbon/carbon_user = loc
+	if(IS_HERETIC(carbon_user) || IS_HERETIC_MONSTER(carbon_user))
+		. = ..()
+		//We need to account for the hood shenanigans, and that way we can make sure items always fit, even if one of the slots is used by the fucking hood.
+		if(suittoggled)
+			to_chat(carbon_user,"<span class='notice'>The light shifts around you making the cloak invisible!</span>")
+		else
+			to_chat(carbon_user,"<span class='notice'>The kaleidoscope of colours collapses around you, as the cloak shifts to visibility!</span>")
+		item_flags = suittoggled ? EXAMINE_SKIP : ~EXAMINE_SKIP
+	else
+		to_chat(carbon_user,"<span class='danger'>You can't force the hood onto your head!</span>")
+
+
+/obj/item/clothing/mask/void_mask
+	name = "Abyssal Mask"
+	desc = "Mask created from the suffering of existance, you can look down it's eyes, and notice something gazing back at you."
+	icon_state = "mad_mask"
+	inhand_icon_state = "mad_mask"
+	w_class = WEIGHT_CLASS_SMALL
+	flags_cover = MASKCOVERSEYES
+	resistance_flags = FLAMMABLE
+	flags_inv = HIDEFACE|HIDEFACIALHAIR
+	var/mob/living/carbon/human/local_user
+
+/obj/item/clothing/mask/void_mask/equipped(mob/user, slot)
+	. = ..()
+	if(ishuman(user) && user.mind && slot == ITEM_SLOT_MASK)
+		local_user = user
+		START_PROCESSING(SSobj,src)
+		if(IS_HERETIC(user) || IS_HERETIC_MONSTER(user))
+			return
+		ADD_TRAIT(src, TRAIT_NODROP, CLOTHING_TRAIT)
+
+/obj/item/clothing/mask/void_mask/dropped(mob/M)
+	. = ..()
+	local_user = null
+	STOP_PROCESSING(SSobj,src)
+	REMOVE_TRAIT(src, TRAIT_NODROP, CLOTHING_TRAIT)
+
+/obj/item/clothing/mask/void_mask/process(delta_time)
+	if(!local_user)
+		return PROCESS_KILL
+
+	if((IS_HERETIC(local_user) || IS_HERETIC_MONSTER(local_user)) && HAS_TRAIT(src,TRAIT_NODROP))
+		REMOVE_TRAIT(src, TRAIT_NODROP, CLOTHING_TRAIT)
+
+	for(var/mob/living/carbon/human/human_in_range in spiral_range(9,local_user))
+		if(IS_HERETIC(human_in_range) || IS_HERETIC_MONSTER(human_in_range))
+			continue
+
+		SEND_SIGNAL(human_in_range,COMSIG_VOID_MASK_ACT,rand(-2,-20)*delta_time)
+		if(DT_PROB(60,delta_time))
+			human_in_range.hallucination += 5
+
+		if(DT_PROB(40,delta_time))
+			human_in_range.Jitter(5)
+
+		if(DT_PROB(30,delta_time))
+			human_in_range.emote(pick("giggle","laugh"))
+			human_in_range.adjustStaminaLoss(10)
+
+		if(DT_PROB(25,delta_time))
+			human_in_range.Dizzy(5)
+
+/obj/item/melee/rune_knife
```

Comment:
So why is this different from the knives you already get? Can't this be handled with the eldritch blade?

Reply:
no, this is not an eldritch blade, eldritch blade inherits the whole marks, enhancments and shit. This is purely for drawing magic circles on the ground.